.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,20 @@
+--
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
+Add any other context or screenshots about the feature request here.
```

Comment:
I am wondering whether a section _Additional context_ is useful. The goal of the template is to guide the creator of the ticket as such that he/she doesn't forget to mention necessary information. I am afraid that this section doesn't provide much guidance.

Reply:
Removed it in https://github.com/inmanta/inmanta/pull/2416/commits/4f10b989b54c0c76c2a13c81416026cef61a37ed